File: bench/Reference/int_array_map.java
public class int_array_map {
    public static void main(String[] args) {
        int[] a = new int[10000];
        for (int i = 0; i < 10000; i++) a[i] = i + 1;
        int[] doubled = new int[10000];
        for (int i = 0; i < 10000; i++) doubled[i] = a[i] * 2;
        long s = 0;
        for (int x : doubled) s += x;
        System.out.println(s);
    }
}

File: bench/Reference/int_array_map.erl
#!/usr/bin/env escript
%% Map (*2) over 1..10000 and sum.

main(_) ->
    Xs = lists:seq(1, 10000),
    Doubled = lists:map(fun(X) -> X * 2 end, Xs),
    io:format("~p~n", [lists:sum(Doubled)]).
